src/main/java/frc/robot/hardware/motor/sparkmax/SparkMaxWrapper.java

```diff
@@ -13,4 +15,8 @@ public double getVoltage() {
 		return getAppliedOutput() * getBusVoltage();
 	}
 
+	public Rotation2d getVelocity() {
```

Comment:
```suggestion
	public Rotation2d getVelocityPerSecond() {
```

Reply:
Why don't you have a `RevolutionsPerSecond` class?

Reply:
"velocity per second" is acceleration,

Reply:
It's not "position" per second, it's movement per second, or more accurately, `RevolutionsPerSecond`, as mentioned above, and as is the standard unit for measuring rates of rotational movement. This could either be the name of the returned type or the function name, though I like having separate types for different units personally.

Reply:
so what should i call it?